File: 2019.1.26 P1422 小玉家的电费.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    int used;
    cin >> used;
    float origin;
    if(used <= 150)
    {
        origin = used * 0.4463;
    }
    else if(used <= 400)
    {
        origin = (used - 150) * 0.4663 + 66.945;
    }
    else
    {
        origin = (used - 400) * 0.5663 + 183.52;
    }
    cout << fixed << setprecision(1) << origin;
    return 0;
}

File: 2019.1.26 P1422 小玉家的电费.py
used = eval(input())
if used <= 150:
    origin = 0.4463 * used
elif used <= 400:
    origin = 0.4663 * (used-150) + 66.945
else:
    origin = 0.5663 * (used-400) + 183.52
result = int((origin * 10)+0.5)/10.0
print(result)